### PR TITLE
do not require Content Security Policy (CSP) style-src 'unsafe-inline'

### PR DESCRIPTION
**Pull request by jelhan**
To not trigger a Content Security Policy (CSP) 'unsafe-inline' violation we have to modify CSSOM directly. This is what jQuery `.css()` method does. Therefore code like `$('<div style="width: 100"');` should be avoided to not force consuming applications to lower CSP rules.

### CODE REVIEW

**Comment by rugk** [comment]
I think this is also much cleaner JS/jQuery code. :+1:

File: dist/jquery.floatThead-slim.js [hunk 1]
```diff
@@ -67,7 +67,15 @@
   //safari 7 (and perhaps others) reports table width to be parent container's width if max-width is set on table. see: https://github.com/mkoryak/floatThead/issues/108
   var isTableWidthBug = function(){
     if(isWebkit) {
-      var $test = $('<div style="width:0px"><table style="max-width:100%"><tr><th><div style="min-width:100px;">X</div></th></tr></table></div>');
+      var $test = $('<div>').css('width', 0).append(
+        $('<table>').css('max-width', '100%').append(
+          $('<tr>').append(
+            $('<th>').append(
+              $('<div>').css('min-width', 100).text('X')
```
Comment (mkoryak) [inline dist/jquery.floatThead-slim.js]: unit missing

File: dist/jquery.floatThead-slim.js [hunk 2]
```diff
@@ -482,15 +514,20 @@
...
+            );
           }
 
           cols = cols.join('');
           cells = cells.join('');
 
           if(createElements){
-            psuedo = psuedo.join('');
-            $fthRow.html(psuedo);
+            $fthRow.append(psuedo);
```
Comment (mkoryak) [inline dist/jquery.floatThead-slim.js]: is appending an array actually any faster than `html` which i assume uses innerHTML? 

I am optimizing for speed vs clarity in most cases here.

Comment (jelhan) [inline dist/jquery.floatThead-slim.js]: `.html('<div style="width: 100%"></div>')` violates CSP 'unsafe-inline'. Therefore refactored to append an array of jQuery elements. Didn't considered performance.

Comment (mkoryak) [inline dist/jquery.floatThead-slim.js]: Ah ok. Will murge soon

Comment (rugk) [inline dist/jquery.floatThead-slim.js]: *merge :smiley:

**Comment by jelhan** [comment]
Thanks!

**Comment by jelhan** [comment]
@mkoryak Do you mind to release a patch version?

**Comment by mkoryak** [comment]
I don't normally release with so little changes, but since you took time to make a PR, here is your new version:

https://github.com/mkoryak/floatThead/releases/tag/1.4.3

**Comment by jelhan** [comment]
Thanks a lot!

**Pull request merged**